gcp/bigquery/postgres_to_bq/bq_create_fhir_tables.sql

```diff
@@ -0,0 +1,214 @@
+DROP TABLE IF EXISTS mimic_fhir.patient;
+CREATE TABLE mimic_fhir.patient (
+  id STRING(36),
+  fhir STRING
```

Comment:
Tried JSON for the `fhir` datatype but it was not pulling into python as a json, so wasn't worth it.

Also tried to `--autodetect` when loading data in but it just created `fhir` as a STRING anyways